### ISSUE TITLE
Improve test case test_create_scaling_dataset

### ISSUE THREAD

**Issue by fluegelk** [Helmholtz-AI-Energy/special-couscous#30]
When testing the scaling dataset, we aim to verify that the different splits (global test set and local train sets) are drawn from the same distribution.
For now, we test this by checking that the feature- and class-wise means of all splits are within one standard deviation of the mean of the global training set. However, this check does not seem very meaningful since all means seem to be close to 0 with stds around 1.

Maybe there is a better way to check that all splits are from the same distribution?
